cocotb/triggers.py

```diff
@@ -161,11 +161,50 @@ def unprime(self):
 class Timer(GPITrigger):
     """Fires after the specified simulation time period has elapsed."""
     def __init__(self, time_ps, units=None):
+        """
+        Args:
+           time_ps: The time value. 
+               This can also be a :class:`fractions.Fraction` or
+               a :class:`decimal.Decimal` instance.
+               Note that despite the name this is not actually in picoseconds
+               but depends on the *units* argument.
+           units (str or None, optional): One of
+               ``None``, ``'fs'``, ``'ps'``, ``'ns'``, ``'us'``, ``'ms'``, ``'sec'``.
+               When no *units* is given (``None``) the timestep is determined by
+               the simulator.
+
+               See Also:
+                   :func:`~cocotb.utils.get_sim_steps`
+
+     Examples:
```

Comment:
This should be indented to the same level as `Args`

Reply:
Tabs vs spaces, don't you love it...